### ISSUE TITLE
Epic – Listings (Seller CRUD)

### ISSUE THREAD

**Issue by remylian** [remylian/FED2-SP2#14]
Implementation of CRUD operations: 

- [x] Create Listing
- [x] Edit my listing
- [x] Delete my listing
- [x] Media Gallery (alt text, handling broken/missing images)
